src/test/kotlin/io/getunleash/polling/AutoPollingPolicyTest.kt

```diff
@@ -89,7 +89,7 @@ class AutoPollingPolicyTest {
         val server = MockWebServer()
         val isCalled = CompletableFuture<Unit>()
         val pollingMode = PollingModes.autoPoll(20) { isCalled.complete(null) }
-        val config = UnleashConfig.newBuilder().proxyUrl(server.url("/").toString()).clientSecret("my-secret").build()
+        val config = UnleashConfig.newBuilder().proxyUrl(server.url("/").toString()).clientKey("my-secret").build()
```

Comment:
The above `key` vs `secret` in variable values applies here too.

Reply:
Fixed in 5f9462e